src/MarDyn.cpp

```diff
@@ -134,7 +134,10 @@ int run_unit_tests(const Values &options, const std::vector<std::string> &args)
 int main(int argc, char** argv) {
 #ifdef ENABLE_MPI
 	MPI_Init(&argc, &argv);
+	int world_rank = 0;
+	MPI_Comm_rank(MPI_COMM_WORLD, &world_rank);
```

Comment:
Maybe this could go to the place where `world_rank` is actually used? Also, I would not assign a value to `int world_rank` first as this is just confusing and will be overwritten anyway.

Reply:
Some compilers or clang tidy might complain if a variable is not initialized. But since the initialization here truly doesn't matter, we could just use default initialization.

```suggestion
	int world_rank{};
	MPI_Comm_rank(MPI_COMM_WORLD, &world_rank);
```